frontend/src/metabase/components/ThinProgressBar.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+
+import "./ThinProgressBar.css";
+
+export default class ThinProgressBar extends Component {
+  static defaultProps = {
+    fill: "currentcolor",
+    className: "loader",
+  };
+
+  render() {
+    var { size, className } = this.props;
```

Comment:
Small thing but I think `const` is more appropriate here based on what we do elsewhere and so that it yells if someone were to mess with those values elsewhere.